layers/meta-resin-genericx86/recipes-kernel/linux/files/realsense_powerlinefrequency_control_fix_linux-yocto_4.4.patch

```diff
@@ -0,0 +1,26 @@
+From e53ff5bf09a54af77f3f2f7c7282e38c33093377 Mon Sep 17 00:00:00 2001
+From: Scott Ware <scott.r.ware@intel.com>
+Date: Mon, 27 Nov 2017 15:27:40 +0000
+Subject: [PATCH 4/4] Fix incomplete PowerLineFrequency enumeration map
+
+Upstream-Status: Backport
```

Comment:
needs also signed-ff by you, Theodor, in the correct order.
See https://github.com/resin-os/resin-intel/blob/master/layers/meta-resin-genericx86/recipes-kernel/linux/files/0001-Add-support-for-Quectel-EC20-modem.patch for example